### ISSUE TITLE
“Follow news about us” 

### ISSUE THREAD

**Issue by antaldaniel** [dataobservatory-eu/openmuse.dataobservatory.eu#24]
– leads back to main (we might want a separate section for that)

I think that we should have at least 3 news items by now:
- We were contracted
- Kick off meeting
- MoU with the Slovak ministry
Curretnly it leads nowhere because I do not have th news

**Comment by antaldaniel** [comment]
Yes, absoultely, as soon as Mark writes up :))

**Comment by antaldaniel** [comment]
No, Isabella took a group photo on the kick off meeting. I would assume that we should have that photo and a blog post about the kick off meeting.

**Comment by jrhyse** [comment]
Ah, got it. Yes I will get that from her.
